NICE.Identity.Authentication.Sdk/API/APIService.cs

```diff
@@ -105,6 +106,42 @@ public async Task<IEnumerable<UserDetails>> FindUsers(IEnumerable<string> nameId
 			}
 			throw new Exception($"Error calling the API. status code: {(int)responseMessage.StatusCode}");
 		}
+
+		/// <summary>
+		/// Refresh users token
+		/// </summary>
+		/// <param name="nameIdentifier"></param>
+		/// <param name="httpClient"></param>
+		/// <returns></returns>
+		public async Task<string> RevokeRefreshTokensForUser(string nameIdentifier, HttpClient httpClient = null)
```

Comment:
could this be refactored to use the new PostToAPI private function please?

https://github.com/nice-digital/identity/blob/master/NICE.Identity.Authentication.Sdk/API/APIService.cs#L103

all the duplicated code in APIService got moved out to a private function as part of ec-831, which has beaten this change to live, so it no longer matches.